### PR TITLE
WIP: c#:New Sample-Credit_ risk_Anomaly_Detection-PCA based anomaly detection

### PR DESCRIPTION
**Pull request by prathyusha12345**
Created new sample CreditRisk Anomaly Detection
-Used German Credit risk Data as the dataset.

### CODE REVIEW

File: samples/csharp/getting-started/AnomalyDetection_CreditRisk/CreditRiskDetection/CreditRiskDetectionConsoleApp/Program.cs [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+            //var peak2 = trainTestData.TrainSet.Preview();
+            //var peak3 = trainTestData.TestSet.Preview();
+            //string[] featureColumnNames = dataView.Schema.AsQueryable().Select(column => column.Name).ToArray();
+
+            // 2. Create a pipeline to prepare your data
+            // 2a. There are 2 types of features in the data. 1.Numerical and 2.Categorical.
+            //Transform Categorical features into Numeric values
+            //Concatenate numeric features and transformed categorical features
+            //Drop Label column from the pipeLine because Anomaly Detection trainer trains on unsupervised data.
+            IEstimator<ITransformer> dataProcessingPipeline = mlContext.Transforms.Concatenate("NumFeatures", nameof(CustomerCreditData.numOfMonths),
```
Comment (shmoradims) [inline samples/csharp/getting-started/AnomalyDetection_CreditRisk/CreditRiskDetection/CreditRiskDetectionConsoleApp/Program.cs]: NumFeatures means number of features for most people. Please use NumericFeatures instead.

File: samples/csharp/getting-started/AnomalyDetection_CreditRisk/CreditRiskDetection/CreditRiskDetectionConsoleApp/DataStructures/CustomerCreditData.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+       public string StatusAndSex { get; set; }
+
+       [LoadColumn(9)]
+       public string gurantors { get; set; }
+
+       [LoadColumn(10)]
+       public float ResidentSince { get; set; }
+      
+       [LoadColumn(11)]
+       public string property { get; set; }
```
Comment (shmoradims) [inline samples/csharp/getting-started/AnomalyDetection_CreditRisk/CreditRiskDetection/CreditRiskDetectionConsoleApp/DataStructures/CustomerCreditData.cs]: Please use the same casing for all features. Either PascalCase or camelCase. Don't mix.

**Comment by CESARDELATORRE** [comment]
@shmoradims - When you have the time, could you take a look to the issue we're having with this sample, here: 
https://github.com/dotnet/machinelearning/issues/3038

File: samples/csharp/getting-started/AnomalyDetection_CreditRisk/CreditRiskDetection/CreditRiskDetectionConsoleApp/Program.cs (hunk 1 above)
Comment (prathyusha12345) [reply samples/csharp/getting-started/AnomalyDetection_CreditRisk/CreditRiskDetection/CreditRiskDetectionConsoleApp/Program.cs]: Changed

File: samples/csharp/getting-started/AnomalyDetection_CreditRisk/CreditRiskDetection/CreditRiskDetectionConsoleApp/DataStructures/CustomerCreditData.cs (hunk 2 above)
Comment (prathyusha12345) [reply samples/csharp/getting-started/AnomalyDetection_CreditRisk/CreditRiskDetection/CreditRiskDetectionConsoleApp/DataStructures/CustomerCreditData.cs]: changed

**Comment by CESARDELATORRE** [comment]
Closing the PR as it is stale

**Pull request closed without merging**